mllib/src/main/scala/org/apache/spark/mllib/regression/StreamingLinearRegressionWithSGD.scala

```diff
@@ -59,6 +59,8 @@ class StreamingLinearRegressionWithSGD private[mllib] (
 
   val algorithm = new LinearRegressionWithSGD(stepSize, numIterations, miniBatchFraction)
 
+  var model: Option[LinearRegressionModel] = None
```

Comment:
I get that the idea is that changes something in the declaration that seems to make it refer to the latest model but I'm having trouble seeing what this would affect, along with the change in the superclass.